bin/rspec

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+begin
+  load File.expand_path('../spring', __FILE__)
+rescue LoadError => e
+  raise unless e.message.include?('spring')
```

Comment:
So it looks like we anticipate some spring LoadError, and are fine with it.  What is the error?

Reply:
I'm not sure, this came from running `spring binstub --all`.